.claude/skills/github/scripts/issue/edit_issue_body.py

```
@@ -0,0 +1,113 @@
+    gh_args = [
+        "gh",
+        "issue",
+        "edit",
+        str(args.issue),
+        "--repo",
+        f"{owner}/{repo}",
+        "--body",
+        body,
+    ]
+
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)